scripts/build-openstack-image.sh

```diff
@@ -54,10 +54,6 @@ if [[ -n "$HTTP_PROXY" ]]; then
         sed -i "/^http_proxy=/d" /etc/environment
         echo "http_proxy=$HTTP_PROXY" >> /etc/environment
     fi
-    if ! grep -q "Acquire::http::Proxy" /etc/apt/apt.conf || ! grep -q "Acquire::http::Proxy \"$HTTP_PROXY\";" /etc/apt/apt.conf; then
-        sed -i "/^Acquire::http::Proxy/d" /etc/apt/apt.conf
-        echo "Acquire::http::Proxy \"$HTTP_PROXY\";" >> /etc/apt/apt.conf
-    fi
```

Comment:
The juju machine already has proxy setting in `/etc/apt/apt.conf.d/95-juju-proxy-settings`.